### ISSUE TITLE
Turbografx cd bin/cue files not working? 

### ISSUE THREAD

**Issue by Gitlab1111** [libretro/RetroArch#10037]
CPU: Intel Haswell Intel© Core™ i5-4670 CPU @ 3.40GHz × 4
GPU: N/A
RAM: 4gb
SSD 60gb

Nightly build 11/4/2009 Lakka OS

Does The core for Turbografx 16 cd's work with bin/cue games because the other ones in ccd seem to work fine except for bin/cue combo files. Logs 

[Retroarch TG16cd.txt](https://github.com/libretro/RetroArch/files/4084175/Retroarch.TG16cd.txt)

**Comment by RobLoach** [comment]
Yes, they work. Was just able to run Golden xe.

**Comment by negativeExponent** [comment]
`[INFO] Using content: /storage/roms/turbografxcd/Exile 2/Exile 2 - Wicked Phenomenon (USA).bin.`

i cant remember when beetle/mednafen is able to directly open .bin/iso files. they always (and should be) opened with the correct cuesheet or toc. so those .info needs to be edited

this should be in the cores section.

**Comment by Gitlab1111** [comment]
I still get nothing but a black screen, I am using another nightly build different from the one I had before yet same results. 
[NB Jan24 2020 t16cd.txt](https://github.com/libretro/RetroArch/files/4110928/NB.Jan24.2020.t16cd.txt)
Here are some logs

**Comment by negativeExponent** [comment]
> I still get nothing but a black screen, I am using another nightly build different from the one I had before yet same results.
> [NB Jan24 2020 t16cd.txt](https://github.com/libretro/RetroArch/files/4110928/NB.Jan24.2020.t16cd.txt)
> Here are some logs
> 
> I am using Bettle PCE Fast, I also had used system 3 and 2 bios even using the US system 3 bios did not change the end result.

- you havent read my reply above? you are trying to load directly .BIN file in PCE which is incorrect. load your game using the .CUE file and your game should would (unless you got a bad dump)

- and this is not a place to report core-related problems


-

**Comment by Gitlab1111** [comment]
When I used an earlier build the roms in ccd format were launching but not the ones in bin/cue. Yes I had launched  the cue files as well. Using a more recent nightly build the bin/cue roms are working but not Exile2. I don't know if its a bad rom but here are the logs for it.


 When I posted the issue I didn't know if it was appropriate to post it here, it does not say anything against it and I see numerous core related posts here.

[T16cd NB Lakka.txt](https://github.com/libretro/RetroArch/files/4111103/T16cd.NB.Lakka.txt)
